index.js

```diff
@@ -0,0 +1,49 @@
+// const { default: axios } = require("axios");
+
+
+const select = document.querySelector("select");
+const form = document.querySelector("form");
+const reviewInput = document.querySelector("#review-input");
+const submittedReviewsUl = document.querySelector("#submitted-reviews-ul");
+const movieTitle = document.querySelector("#title-h3");
+const releaseYear = document.querySelector("#release-year-p");
+const movieDescription = document.querySelector("#description-p");
+
+const createMovieSelector = async () => {
+  try {
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+
+    res.data.forEach((movie) => {
+      const option = document.createElement("option");
+      option.innerText = movie.title;
+      select.appendChild(option);
```

Comment:
we could also use the `option.value` property to save the id of the current movie or we can save the url of that movie `movie.url` to the value property as well.